ceph.spec.in

```diff
@@ -27,7 +27,7 @@
 # no gperftools/tcmalloc on s390(x)
 %bcond_with tcmalloc
 %endif
-%bcond_with lowmem_builder
+%bcond_without lowmem_builder
```

Comment:
This means that by default the build will have the `lowmem_builder` enabled. I'm not sure what `lowmem_builder` really means. Does this mean that we cannot build Ceph in OBS without restraining the memory resources used?